frontend/src/app/spot/styles/sass/components/list.sass

```diff
@@ -29,7 +29,6 @@
         @include spot-focus
 
       &:hover
-        color: var(--content-link-color)
```

Comment:
Hi @HDinger, thanks for remarks.
I checked Github, while hovering over list items, the color is not changed both in normal and high-contrast modes. 
I thought maybe we want it to look like Github.

Reply:
Hi @HDinger ,
I asked, in normal mode, it will be kept as it is now (color change on hover effect). In high-contrast mode, the color won't change while hovering over items.